app/main.py

```diff
@@ -1,5 +1,72 @@
 class Cargo:
-    def __init__(self, weight: int) -> None:
+    def __init__(self, weight: int | float) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int | float,
+            coords: list = None
```

Comment:
The value could be both types, so use an annotation like this. The same in other similar cases


```suggestion
    def __init__(
            self,
            name: str,
            weight: int | float,
            coords: list | None= None
```